homeassistant/components/sensor/__init__.py

```diff
@@ -72,6 +73,7 @@
     DEVICE_CLASS_CURRENT,  # current (A)
     DEVICE_CLASS_DATE,  # date (ISO8601)
     DEVICE_CLASS_ENERGY,  # energy (kWh, Wh)
+    DEVICE_CLASS_FREQUENCY,  # frequency (Hz, MHz, GHz)
```

Comment:
What about "kHz" (kilohertz)? Only Hz and KHz exists as scales for Frequency in the Z-Wave specifications, see Z-Wave JS -> https://github.com/zwave-js/node-zwave-js/discussions/3695

Reply:
kHz is not (yet?) part of the standard units in const.py:

https://github.com/home-assistant/core/blob/5370dd81226853c7c8e6b2c5b7310c38a3e0948a/homeassistant/const.py#L472-L475

Reply:
hmm, can "kHz" that be just added as an additional standard unit in this PR or is a new architecture discussion and decision needed? https://github.com/home-assistant/architecture/discussions/

Reply:
We can list we would allow it in the documentation/comments, but we can't add it to the constants, as that would introduce dead code.

Reply:
```suggestion
    DEVICE_CLASS_FREQUENCY,  # frequency (Hz, kHz, MHz, GHz)
```